pipelines/_ingredients/cpSkyLSST.yaml

```diff
@@ -0,0 +1,29 @@
+description: Sky frame generation pipeline definition for LSST isr task.
+tasks:
+  cpSkyIsr:
+    class: lsst.ip.isr.IsrTaskLSST
+    config:
+      connections.outputExposure: "cpSkyIsrExp"
+      # TODO: Add deferred charge and brighter-fatter pipelines
+      doBrighterFatter: false
```

Comment:
Does this TODO match the one above's missing ticket number?
In any case, I don't think the impact of the BFE and CTI will be that large on sky frames, as they have lots of smoothing.

Reply:
Would like to have everything consistent, and this is the same TODO.